### PR TITLE
load about

### PR DESCRIPTION
**Pull request by lyvanminh**

### CODE REVIEW

**Comment by lyvanminh** [comment]
screenshot from 2018-10-01 10-21-37

**Comment by ndviet2307** [comment]
OK

**Comment by lyvanminh** [comment]
ready

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.page(params[:page])
+                 .per Settings.posts_controller.per_page
+  end
+
+  def show
+    @post = Post.find_by id: params[:id]
```
Comment (DuongNT1512) [inline app/controllers/posts_controller.rb:8]: báo lỗi nếu k tìm thấy

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<% provide :title, t(".title") %>
+
+<div class="tintuc-content">
+  <div class="content-news">
+    <div class="container">
+      <h1 class="titlepage"><%= t ".title" %></h1>
+      <div class="new-title">
+        <div class="entry-content notopmargin">
+        <% @posts.each do |post| %>
+        <div class="post">
```
Comment (DuongNT1512) [inline app/views/posts/index.html.erb]: indent

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<% provide :title, t(".title") %>
+
+<div class="tintuc-content">
+  <div class="content-news">
+    <div class="container">
+      <h1 class="titlepage"><%= t ".title" %></h1>
+      <div class="new-title">
+        <div class="entry-content notopmargin">
+        <% @posts.each do |post| %>
```
Comment (DuongNT1512) [inline app/views/posts/index.html.erb]: dunfg render collection nhé em

File: app/views/posts/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        </div>
+        <div class="nguontin" style="text-align: right;">
+          <strong><%= @post.user.name %></strong>
+        </div>
+        <div class="clear"></div>
+      </div>
+      <div class="entry-content lienquan">
+        <h3><%= t ".relate_posts" %></h3>
+        <div class="row">
+          <% @relate_posts.each do |relate_post| %>
```
Comment (DuongNT1512) [inline app/views/posts/show.html.erb]: tuowng tự

File: app/views/posts/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+            <div class="post-lienquan">
+              <div class="post-image">
+                <%= link_to image_tag("tintuc.jpg", class: "img-responsive"), post_path(relate_post) %>
+              </div>
+              <div class="post-title-lq">
+                <h2><a href="#"><%= relate_post.title %></a></h2>
+              </div>
+              <ul class="post-date">
+                <li><i class="icon-calendar3"><%= l @post.created_at, format: :short %></i></li>
+                <li><i class="icon-user">/</i><%= @post.user.name %></li>
```
Comment (DuongNT1512) [inline app/views/posts/show.html.erb]: dunfg delegate nhé

**Comment by DuongNT1512** [comment]
commented

**Comment by ndviet2307** [comment]
OK

**Comment by lyvanminh** [comment]
ready

File: app/views/posts/_relate_post.html.erb [hunk 6]
```diff
@@ -0,0 +1,18 @@
+<div class="col-md-6 news-lienquan">
+  <div class="post-lienquan">
+    <div class="post-image">
+      <%= link_to image_tag("tintuc.jpg", class: "img-responsive"), post_path(relate_post) %>
+    </div>
+    <div class="post-title-lq">
+      <h2><a href="#"><%= relate_post.title %></a></h2>
```
Comment (DuongNT1512) [inline app/views/posts/_relate_post.html.erb]: link_to nhé em

**Comment by DuongNT1512** [comment]
commented

**Comment by lyvanminh** [comment]
ready

**Pull request merged**